src/parser/Parser.java

```diff
@@ -69,12 +70,21 @@ public float getDailyWorkHours(String data) throws WrongFormatException, ParseEx
 					final Date date1 = format.parse(processedFile.getRows().get(i).getStartHour());
 					final Date date2 = format.parse(processedFile.getRows().get(i).getEndHour());
 					final float difference = TimeUnit.MILLISECONDS.toMinutes(date2.getTime() - date1.getTime());
-					oreLavorate = oreLavorate + difference;
+					minLavorati = minLavorati + difference;
 				}
 			}
 		} catch (final ParseException e) {
 			throw new WrongFormatException("Error: ",e);
 		}
+		
+		oreLavorate = (int) minLavorati / 60;
+		minLavorati = minLavorati % 60;
+		if (minLavorati != 10) {
+			oreLavorate = oreLavorate + minLavorati / 100;
+		} else {
+			oreLavorate = oreLavorate + minLavorati / 10;
+		}
+		
 		return oreLavorate / 60;
```

Comment:
Non penso serva più / 60 visto che la conversione è già stata fatta
```suggestion
		return oreLavorate;
```

Reply:
Infatti era un errore della sovrapposizione dei metodi, essendo i 3 quasi uguale.